### DISCUSSION TITLE
Is it possible to make FnLock feature to work only with internal keyboard?

### DISCUSSION THREAD

**Discussion by sshanahin** [seerge/g-helper#2220] [General]
I have a G14 2023 laptop
My case -- I usually work using external display and external keyboard (MX Keys) and I need the F1-F12 keys in this case 
but I also often use the laptop as standalone for enertainment ( gaming, watching films etc) -- and in this case Media keys are more convenient. 
So far so good -- can just turn on FnLock in standalone mode and turn it off -- in desktop mode

But it would be useful to have an ability to have FnLock always enabled for internal keyboard to have an ability to change Aura modes, Fan profiles and internal screen brightness just by pressing corresponding butttons on laptop keyboard, keeping the external keyboard working in usual way (F1-F12) and it's own set of media keys not available from laptop keyboard

**Comment by seerge** [comment]
@sshanahin hello, unfortunately - not :)

It's a "software" fn-lock, app just registers F1 .. F11 as own "hotkeys" in windows (when it's on), and runs corresponding actions when they are pressed. 

https://stackoverflow.com/questions/2450373/set-global-hotkeys-using-c-sharp
